app/forms/FormProvider.scala

```diff
@@ -55,8 +55,8 @@ abstract class FormProvider[E, A](fieldMapping: (E, Seq[String]) => FieldMapping
     Form(
       mapping[(A, A, A), A, A, A](
         "value.1" -> fieldMapping(errors1, args),
-        "value.2" -> fieldMapping(errors2, args),
-        "value.3" -> fieldMapping(errors3, args)
+        "value.2" -> fieldMapping(errors2, args.drop(1)),
+        "value.3" -> fieldMapping(errors3, args.drop(2))
```

Comment:
This breaks validation messages in `AmountOfTheLoanController`.
![image](https://github.com/user-attachments/assets/e36ff1fe-52d3-40e5-973c-adeb92392286)


I suggest breaking down args parameter into three separate ones:
```
    args1: Seq[String],
    args2: Seq[String],
    args3: Seq[String]
```


(similar in above method with two errors for consistency,  even though that doesn't seem to break anything)